introduction/Java_Currency_Formatter/Solution.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Locale;
+import java.util.Locale.Builder;
+import java.text.NumberFormat;
+import java.util.Scanner;
+
+public class Solution {
+  
+  public static final Locale localIndia = new Locale("en", "IN"); 
+  
+  private static String getCurrency(Locale currency, double priceToConvert) {
+    NumberFormat numberFormat = NumberFormat.getCurrencyInstance(currency);
+    String paymentInTheProvidedCurrency = numberFormat.format(priceToConvert);
```

Comment:
merge this and next line. No need for creating a variable just to return it in the next line.